File: leetcode/27. Remove Element/27._Remove_Element_1.java
public class Solution {
    public int removeElement(int[] nums, int val) {
        int left = 0, right = nums.length-1;
        int size = nums.length;
        while( left < size ) {
            if( nums[left] == val) {
                int temp = nums[left];
                nums[left] = nums[right];
                right--;
                size--;
            }
            else {
                left++;
            }
        }
        return size;
    }
}

File: leetcode/27. Remove Element/27._Remove_Element_1.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        count = nums.count(val);
        counter = 0
        while counter != count:
            nums.remove(val);
            counter += 1;
        return len(nums)
        
        